CONTRIBUTING.md references a missing `make test` target

## Description

`CONTRIBUTING.md` instructs contributors to run `make test` under **Running Tests and Linting**, but the current `Makefile` does not define a `test` target.

## Reproduction

From a clean checkout with the development environment installed:

```shell
make test
```

produces:

```text
make: *** No rule to make target `test'.  Stop.
```

The repository's CI currently runs `make test-base` first and then invokes the source-specific targets such as `make test-caiso`, `make test-ercot`, and others separately.

## Expected behavior

The documented command should correspond to a supported test workflow. Would you prefer to:

1. update `CONTRIBUTING.md` to document `make test-base` and the source-specific targets, or
2. add an aggregate `make test` target?

I reproduced this while setting up the project locally and would be happy to submit a focused PR once the preferred behavior is confirmed.

--- a/Makefile
+++ b/Makefile
@@ -10,6 +10,20 @@ PYTEST_CMD := uv run pytest -s -vv -n auto --reruns 5 --reruns-delay 3 --duratio
 NOT_SLOW := -m "not slow"
 UNIT_ONLY := -m "not integration"
 
+.PHONY: test
+test: test-base \
+	test-aeso \
+	test-caiso \
+	test-ercot \
+	test-isone \
+	test-miso \
+	test-nyiso \
+	test-pjm \
+	test-spp \
+	test-eia \
+	test-ieso \
+	test-misc
+
 .PHONY: test-base
 test-base:
 	$(PYTEST_CMD) gridstatus/tests/test_*.py --ignore=gridstatus/tests/source_specific/

